ops/jenkins/bfd-mgmt-synthea-generation-build/build-push.sh

```diff
@@ -1,11 +1,21 @@
 #!/usr/bin/env bash
 set -eo pipefail
 
-# The build context's directory is the directory of this script, so always ensure that the context
-# is set to the proper directory regardless of where this script is called
-BUILD_CONTEXT_ROOT_DIR=$(readlink -f "$(dirname "${BASH_SOURCE[0]}")")
+# BUILD_CONTEXT_ROOT_DIR will return the directory where this script is located, and CONTEXT_DIR will be the
+# directory of the Docker build context. This way, this script can be called from _any_ directory and
+# there will be no issues
+BUILD_CONTEXT_ROOT_DIR="$(readlink -f "$(dirname "${BASH_SOURCE[0]}")")"
 readonly BUILD_CONTEXT_ROOT_DIR
 
+REPO_ROOT="$(git rev-parse --show-toplevel)"
+CONTEXT_DIR="${REPO_ROOT}/ops/ccs-ops-misc/synthetic-data/scripts/synthea-automation"
+readonly CONTEXT_DIR
+
+# We want this script to be included in the Docker image while still living inside the build context,
```

Comment:
I legitimately have no idea if this is the best way of achieving what I'm trying to achieve, but it didn't sit well with me having a docker dependent script sitting with the synthea scripts. Open to any other ideas